uavcan/register/384.Access.1.0.uavcan

```diff
@@ -64,19 +64,73 @@
+# they enable enhanced introspection capabilities and simplify device configuration and diagnostics.
+#
+#   REGISTER NAME PATTERN                               TYPE            FLAGS                   RECOMMENDED DEFAULT
+# =====================================================================================================================
+#
+#   uavcan.node.id                                      natural16       mutable, persistent     65535 (unset/PnP)
+#
+# Contains the node-ID of the local node. Values above the maximum valid node-ID for the current transport
+# indicate that the node-ID is not set; if plug-and-play is supported, it will be used by the node to obtain an
+# automatic node-ID. Invalid values other than 65535 should be avoided for consistency.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.node.description                             string          mutable, persistent     (empty)
+#
+# User/integrator-defined, human-readable description of this specific node.
+# This is intended for use by a system integrator and should not be set by the manufacturer of a component.
+# For example: on a quad-rotor drone this might read "motor 2" for one of the ESC nodes.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.cookie                                       string          mutable, persistent     (empty)
+#
+# Arbitrary data stored by an external autoconfiguration authority (e.g., a mission computer, an auto-configuration
+# utility, etc.). This data is never to be used by the node itself; instead, the node merely provides raw data
+# storage space.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.pub.PORT_NAME.id                             natural16       mutable, persistent     65535 (unset, invalid)
+#   uavcan.sub.PORT_NAME.id                             ditto           ditto                   ditto
+#   uavcan.cln.PORT_NAME.id                             ditto           ditto                   ditto
+#   uavcan.srv.PORT_NAME.id                             ditto           ditto                   ditto
+#
+# Publication/subscription/client/server port-ID, respectively. These registers are configured by the system integrator
+# or an autoconfiguration authority when the node is first connected to a network.
+#
+# The "PORT_NAME" defines the human-friendly name of the port, which is related to the corresponding function
+# or a network service supported by the node. The name shall match the following POSIX ERE expression:
+#
+#   [a-zA-Z_][a-zA-Z0-9_]{0,34}
+#
+# The names are defined by the vendor of the node. The user/integrator is expected to uderstand their meaning and
```

Comment:
```suggestion
# The names are defined by the vendor of the node. The user/integrator is expected to understand their meaning and
```

Reply:
My spell checker is malfunctioning again.